js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
+ *
+ * @param {Object} mappedData The default mappedData object.
+ * @param {string} mappedData.title The processed title.
+ * @param {string} mappedData.url The baseUrl with the slug.
+ * @param {string} mappedData.description The processed description.
+ *
+ * @returns {Object} The new mappedData object.
+ */
+const mapDataToPreview = function( mappedData ) {
+	mappedData.url = mappedData.url.replace( /\s/g, "-" );
```

Comment:
It might be wise to handle other space-like characters as well. What happens in the old snippet preview?

Reply:
Going to check the difference out a bit and create issues if needed.